workspaces/ui-v2/src/optic-components/documentation/PathParameters.tsx

```diff
@@ -40,7 +40,7 @@ export function PathParametersViewEdit(props: PathParametersViewEditProps) {
             name={param.pathComponentName}
             shapes={[alwaysAString]}
             depth={0}
-            // @nic todo
+            // TODO - implement when query params are added
```

Comment:
I believe @acunniffe said there was a new resolver to use here? If so, I can update this in this PR (otherwise I'll follow up with this)

Reply:
Ah this isn't query parameters - this is path parameters - I'll update this - but I'm going to again open a separate PR to tackle this work

Reply:
sorry for typing the wrong thing in that comment